src/eureka/S5_lightcurve_fitting/models/BatmanModels.py

```diff
@@ -179,9 +181,12 @@ def __init__(self, **kwargs):
         self.compute_ltt = (np.all(np.in1d(ltt_params, self.paramtitles))
                             and 'Rs' in self.parameters.dict.keys())
         if self.compute_ltt:
+            if self.multwhite:
+                # Compute ltt correction for each white LC
+                self.compute_ltt_once = False
```

Comment:
This check shouldn't really need to be here. If someone is doing a `multwhite` fit then they should typically be either sharing or fixing their orbital parameters. But I can imagine a world where one might want to allow t0 to vary with wavelength (e.g. limb-asymmetry), and this check would not allow that possibility. Granted, ltt only matters for eclipses while to shifts only really matter of transits, but maybe someone is working with a phase curve and is modelling both simultaneously.

Relatedly though, such a check was apparently already baked into the code in a less conspicuous place:
https://github.com/kevin218/Eureka/blob/533cb23a18b0c28a1556fb5ed6c1bf39fa607d22/src/eureka/S5_lightcurve_fitting/models/BatmanModels.py#L279-L284

I'd vote we remove the checks for `self.multwhite` on both places

Reply:
I was thinking of the instance where a planet has TTVs, which means you'd want to fit a transit time for each visit.  Maybe this would be less confusing if we allowed the user to decide.  We could have a parameter in the ECF that sets `compute_ltt_once` to True or False.  What do you think?

Reply:
Wouldn't that then be redundant with allowing t0 or period to vary as a function of wavelength (`free`) instead of setting them to `fixed` or `shared`?

Reply:
Point taken.

Ok, I reverted back to the old version and, after some testing, it seems to work just fine.

Reply:
Should we remove the less conspicuous `self.multwhite` check from the following line as well?
https://github.com/kevin218/Eureka/blob/533cb23a18b0c28a1556fb5ed6c1bf39fa607d22/src/eureka/S5_lightcurve_fitting/models/BatmanModels.py#L279-L284